test/performance/vectorization/vectorPragmas/forallInStandaloneForeach.chpl

```diff
@@ -0,0 +1,29 @@
+module iters {
+  iter myiter(nn: int, nt: int) {
+    foreach i in 0..#nt {
+      foreach j in i*nn..#nn {
+        yield j;
+      }
+    }
+  }
+
+  // coforall loop in standalone should NOT get vector pragma
```

Comment:
This comment is not making sense to me. Maybe this and the next should just say the forall should get a vector pragma?

Reply:
This test is based on `forallInStandalone.chpl` in the same directory, which also has the same comment :)

I think it is referring to the `coforall` and `for` created while lowering the forall inside the standalone. Or at least, that's how I read it.